docs/desktop/DeveloperGuide.md

```diff
@@ -88,6 +90,7 @@ The SDK uses [Isolated storage](https://learn.microsoft.com/en-us/dotnet/standar
   ```csharp
   var accessToken = await authenticator.TokenProvider.RetrieveToken();
   ```
+> **_NOTE:_** If persistance storage of the SDK is used by the application, tokens are refreshed by the SDK when they expire. Application need not worry about refreshing tokens on expiry.
```

Comment:
Instead adding as note here, pls include the message in the description of this method

Reply:
Mention like "Token lifetime and refresh are handled automatically. Application need not worry about refreshing tokens on expiry"